engine.py

```diff
@@ -181,14 +187,16 @@ def execute_command(self, cmd_key, args):
                 if (
                     tank.util.is_linux()
                     and os.environ.get("KDE_FULL_SESSION") is not None
+                    and importer.qt_version_tuple[0] == 4
```

Comment:
I think `if not engine.has_qt5:` will do the trick here.

Unfortunately, `if engine.has_qt4:` is useless here as it does not properly test for Qt4, it only checks if QtGui is available, which is true for Qt5 as well...

Doing `not qt5` will be an issue when we add support for Qt6.

Another option would be to modify the core so we can get the current version of Qt from Qt itself.

The [QtImporter](https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/util/qt_importer.py#L121) already extracts the information, but unfortunately [the engine](https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/platform/engine.py#L171) discards it.

Maybe `sgtk.platform.qt.__version__` and `sgtk.platform.qt.__version_info__` should contain the string and tuple version of the Qt module?

Then you could do something as simple as 

`if sgtk.platform.qt.__version_info__[0] == 4:` in this branch.

Reply:
I really don't know how I managed to turn this into a complex thing. The version info (Qt version) I need is on the `QtCore` class. 
I was initially thinking I needed the PySide version which would have been `PySide.__version__` but I don't need that at all, and then came up with a complex solution for something that should have been super simple.

I've fixed it by doing:
`QtCore.__version_info__[0] == 4`

Reply:
Ah, even simpler, but does it work with PyQt4?

Reply:
I know that PyQt5.QtCore.__version_info__ does not exist, maybe it's the same for PyQt4. Unfortunately it seems I can't install it on my mac anymore.

Reply:
`QtCore.qVersion()[0]` == "4" will do the trick! It works on PyQt4, PySide and PySide2!